style.css

```diff
@@ -0,0 +1,11 @@
+body {
+    background-image: url("https://elements-video-cover-images-0.imgix.net/files/3a6e02dc-2769-456d-a9d7-248ffe6d4005/inline_image_preview.jpg?auto=compress%2Cformat&fit=min&h=394&w=700&s=38a9dc8d5b2e88fda977f4c701b2f620");
+}
+
+input {
+    margin-top: 1cm;
+}
+
+ul {
+    list-style-type: none;
+}
```

Comment:
Try to put the "Add word" button on the same line with the text filed for the input